doc-content/enterprise-only/installation/about-ba-con.adoc

```diff
@@ -0,0 +1,35 @@
+[id='about-ba-con_{context}']
+
+= {PRODUCT} versioning
+
+{PRODUCT} versions are designated with a numerical `Major.Minor.Patch` format, such as `7.4.1`. In this example, the major release is `7.x.x`, the minor release is `7.4.x`, and the patch release is `7.4.1`. Major releases often require data migration, while minor release upgrades and patch updates are typically managed with update tools provided with the {PRODUCT} release artifacts.
+
+The following are the general types of releases for {PRODUCT}:
+
+Major release migrations::
+
+Major releases of {PRODUCT} include substantial enhancements, security updates, bug fixes, and possibly redesigned features and functions. Data migration is typically required when an application is moved from one major release to another major release, such as from {PRODUCT_OLD} 6.4.x to {PRODUCT} 7.0. Automated migration tools are often provided with new major versions of {PRODUCT} to facilitate migration, but some manual effort is likely required for certain data and configurations. The supported migration paths are specified in product announcements and documentation.
++
+For example migration instructions, see
+ifdef::PAM[]
+link:https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.0/html/migrating_from_red_hat_jboss_bpm_suite_6.4_to_red_hat_process_automation_manager_7.0/[_Migrating from {PRODUCT_OLD} 6.4 to {PRODUCT} 7.0_].
+endif::[]
+ifdef::DM[]
+link:https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.0/html-single/migrating_from_red_hat_jboss_brms_6.4_to_red_hat_decision_manager_7.0/[_Migrating from {PRODUCT_OLD} 6.4 to {PRODUCT} 7.0_].
+endif::[]
+
+Minor release upgrades::
+
+Minor releases of {PRODUCT} include enhancements, security updates, and bug fixes. Data migration may be required when an application is moved from one minor release to another minor release, such as from {PRODUCT} {PRODUCT_VERSION}.x to 7.5. Automated update tools are often provided with both patch updates and new minor versions of {PRODUCT} to facilitate updating certain components of {PRODUCT}, such as {CENTRAL}, {KIE_SERVER}, and the {HEADLESS_CONTROLLER}. Other {PRODUCT} artifacts, such as the {DECISION_ENGINE} and standalone {CENTRAL}, are released as new artifacts with each minor release and you must re-install them to apply the update.
```

Comment:
"{PRODUCT} {PRODUCT_VERSION}.x to 7.5." renders as "7.5.x to 7.5."

Should this be "7.5.x to 7.6"?